### PR TITLE
Implement CI for checking acquisition parameters

### LINKED ISSUE #1: Implement CI testing for BIDS

### PR DESCRIPTION
**Pull request by alexfoias**
The current implementation is using pybids and it checks for TR,TE,FlipAngle and InversionTime (for GE) for T1w. The optimal acquisition parameters from the protocol can be found under `code/specs.json` 

Report from travis: https://travis-ci.com/github/spine-generic/data-single-subject/jobs/359953744

Not sure if we should change the exit code for travis if the script founds warnings.

The current CI is safelisted only for the `af/acq_param_ci` branch. To be changed when ready to merge to master.

Fixes #1

### CODE REVIEW

File: code/params_checker.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    else:
+       logging.warning('Missing Manufacturer in json sidecar')
+
+#Print WARNING log
+if path_warning_log :
+    file = open(path_warning_log, 'r')
+    lines = file.read().splitlines()
+    file.close()
+    for line in lines:
+        print(line)
```
Comment (jcohenadad) [inline code/params_checker.py]: add line at EOF

Comment (alexfoias) [inline code/params_checker.py]:
```suggestion
        print(line)
        
```

**Comment by jcohenadad** [comment]
> Not sure if we should change the exit code for travis if the script founds warnings.

i would not exit with errors-- there will always be warning (like on openneuro) and the only purpose of this CI is to display them. At some point we could think of more rigorous error tests (e.g. if a param is missing, if BIDS structure is not respected, etc.)

**Comment by jcohenadad** [comment]
https://travis-ci.com/github/spine-generic/data-single-subject/jobs/359953744#L236

FA=20deg for the T1w mprage? are you sure about this?

**Comment by jcohenadad** [comment]
https://travis-ci.com/github/spine-generic/data-single-subject/jobs/359953744#L236
can you show TE in ms?  same for TR

**Comment by jcohenadad** [comment]
https://travis-ci.com/github/spine-generic/data-single-subject/jobs/359953744#L238

we need to have tolerance here. i would suggest a tolerance of 0.1ms for TE

**Comment by alexfoias** [comment]
I migrated the CI params to  https://github.com/spine-generic/spine-generic

TE is given in ms.
TR is given in s.

I'll add the tolerance of 0.1ms for TE.

**Comment by alexfoias** [comment]
I also fixed the FA for Trio and Verio.

**Review by jcohenadad — APPROVED** [review]
awesome contribution 👍

**Pull request merged**